not-supported/README.md

```diff
@@ -0,0 +1,12 @@
+# What is this subfolder about?
+This subdirectory of the Nextcloud VM contains scripts that are not yet ready for getting into the Release version.
+
+## Can I help?
+Yes, of course! :)
+Although some things in those scripts might not be 100% ready, we would love to hear your feedback regarding those.
+Feedback is especially welcome, if you would love to have some features that those script brings in the Release version of the NcVM.
+So please report back! 🚀
+
+### How to run this inside my NcVM?
+Currently there is only one script inside this folder. The command to get and run this script is the following:<br>
+`wget https://raw.githubusercontent.com/nextcloud/vm/master/not-supported/smbserver.sh && sudo bash smbserver.sh`
```

Comment:
@enoch85 What do you think about this readme file?

Also, could this be added as a section to the main-readme file as well?

Reply:
Nice README, but not for the main one. 

Maybe add a link to the folder under "Support the development"

Reply:
okay

Reply:
done